test/Bridges/Constraint/slack.jl

```diff
@@ -324,6 +324,18 @@ function test_runtests()
         y <= 2.0
         """,
     )
+    MOI.Bridges.runtests(
+        MOI.Bridges.Constraint.ScalarSlackBridge,
+        """
+        variables: x
+        ScalarNonlinearFunction(log(x)) <= 2.0
+        """,
+        """
+        variables: x, y
+        ScalarNonlinearFunction(log(x) - y) == 0.0
```

Comment:
It's nice to see it working through the bridges with just a few `operate` methods implemented

Reply:
Yeah. The problem is that it's hard to know which `operate` methods are needed :smile:

Reply:
Follow the `MethodError`s :laughing: